exchanges/graphcache/src/cacheExchange.test.ts

```diff
@@ -206,7 +218,7 @@ describe('data dependencies', () => {
     const firstDataTwo = result.mock.calls[1][0].data;
     expect(firstDataOne).not.toBe(firstDataTwo);
     expect(firstDataOne.author).not.toBe(firstDataTwo.author);
-    expect(firstDataOne.unrelated).toBe(firstDataTwo.unrelated);
+    //expect(firstDataOne.unrelated).toBe(firstDataTwo.unrelated);
```

Comment:
It looks like currently this stopped working due to the change which seems peculiar to me as we have actually only changed the logic to use `queryResult.data` which should be equal on the subsequent invocation 😅